[Bug] Unsupported FileSystem s3

### Search before asking

- [X] I had searched in the [issues](https://github.com/streamxhub/streamx/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened
1、
Execution Mode： yarn application 

2、
Flink Home:    /opt/flink/soft/flink-1.12.5
Flink Conf:
state.checkpoints.dir: s3://bgapps/tmp/flink/checkpoints2
state.savepoints.dir: s3://bgapps/tmp/flink/checkpoints2

3、
submit application

### StreamX Version

streamx-1.2.3



### Java Version

1.8

### Flink Version

flink-1.12.5

### Scala Version of Flink

scala-2.11

### Error Exception

```log
Caused by: org.apache.flink.core.fs.UnsupportedFileSystemSchemeException: 
Could not find a file system implementation for scheme 's3'. The scheme is directly supported by Flink through the following plugins: flink-s3-fs-hadoop, flink-s3-fs-presto. Please ensure that each plugin resides within its own subfolder within the plugins directory
```


### Screenshots

![image](https://user-images.githubusercontent.com/17021577/175531452-458256f6-0f25-4f86-938e-379bb8826a52.png)
![image](https://user-images.githubusercontent.com/17021577/175530567-02169ba0-f32f-4197-9cbc-7e8679655534.png)
![image](https://user-images.githubusercontent.com/17021577/175530617-66705a6f-89cc-4586-b483-2aeb11c91d8f.png)


### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/streamx-plugin/streamx-flink-submit/streamx-flink-submit-api/src/main/scala/com/streamxhub/streamx/flink/submit/bean/DeployRequest.scala
+++ b/streamx-plugin/streamx-flink-submit/streamx-flink-submit-api/src/main/scala/com/streamxhub/streamx/flink/submit/bean/DeployRequest.scala
@@ -56,6 +56,7 @@ case class DeployRequest(flinkVersion: FlinkVersion,
       flinkName,
       flinkHome,
       flinkLib = s"$flinkHdfsHome/lib",
+      flinkPlugins = s"$flinkHdfsHome/plugins",
       flinkDistJar = FlinkUtils.getFlinkDistJar(flinkHome),
       appJars = workspace.APP_JARS,
       appPlugins = workspace.APP_PLUGINS

--- a/streamx-plugin/streamx-flink-submit/streamx-flink-submit-api/src/main/scala/com/streamxhub/streamx/flink/submit/bean/SubmitRequest.scala
+++ b/streamx-plugin/streamx-flink-submit/streamx-flink-submit-api/src/main/scala/com/streamxhub/streamx/flink/submit/bean/SubmitRequest.scala
@@ -154,6 +154,7 @@ case class SubmitRequest(flinkVersion: FlinkVersion,
       flinkName,
       flinkHome,
       flinkLib = s"$flinkHdfsHome/lib",
+      flinkPlugins = s"$flinkHdfsHome/plugins",
       flinkDistJar = FlinkUtils.getFlinkDistJar(flinkHome),
       appJars = workspace.APP_JARS,
       appPlugins = workspace.APP_PLUGINS
@@ -191,6 +192,7 @@ case class SubmitRequest(flinkVersion: FlinkVersion,
  * @param flinkHome
  * @param flinkDistJar
  * @param flinkLib
+ * @param flinkPlugins
  * @param appJars
  * @param appPlugins
  * #TODO: className provisional
@@ -199,5 +201,6 @@ case class HdfsWorkspace(flinkName: String,
                          flinkHome: String,
                          flinkDistJar: String,
                          flinkLib: String,
+                         flinkPlugins: String,
                          appJars: String,
                          appPlugins: String)

